apps/web/src/pages/Profile/AccountSettings/NotificationSettings.tsx

```diff
@@ -0,0 +1,262 @@
+                  value={value}
+                  inCheckList
+                />
+              );
+            })}
+          </div>
+        </div>
+      </Field.Fieldset>
+    </Field.Wrapper>
+  );
+};
+
+interface NotificationSettingsProps {
+  ignoredEmailNotifications: NotificationFamily[];
+  ignoredInAppNotifications: NotificationFamily[];
+}
+
+const UpdateIgnoredNotifications_Mutation = graphql(/* GraphQL */ `
+  mutation updateIgnoredNotifications(
+    $ignoredEmailNotifications: [NotificationFamily]
+    $ignoredInAppNotifications: [NotificationFamily]
+  ) {
+    updateIgnoredNotifications(
+      ignoredEmailNotifications: $ignoredEmailNotifications
+      ignoredInAppNotifications: $ignoredInAppNotifications
+    ) {
+      id
+      ignoredEmailNotifications
+      ignoredInAppNotifications
+    }
+  }
+`);
+
+const NotificationSettings = ({
+  ignoredEmailNotifications,
+  ignoredInAppNotifications,
+}: NotificationSettingsProps) => {
+  const intl = useIntl();
+  const [, executeMutation] = useMutation(UpdateIgnoredNotifications_Mutation);
+
+  const methods = useForm<FormValues>({
+    defaultValues: dataValuesToFormValues({
+      ignoredEmailNotifications,
+      ignoredInAppNotifications,
+    }),
+  });
+
+  const notificationOptions = (id: string): CheckboxOption[] => [
+    {
+      value: "email",
+      label: (
+        <span data-h2-visually-hidden="base(invisible)">
+          {intl.formatMessage(commonMessages.email)}
+        </span>
+      ),
+      disabled: id === "systemMessages",
+    },
+    {
+      value: "inApp",
+      label: (
+        <span data-h2-visually-hidden="base(invisible)">
+          {intl.formatMessage(commonMessages.inApp)}
+        </span>
+      ),
+      disabled: id === "systemMessages",
+    },
+  ];
+
+  const { handleSubmit } = methods;
+
+  const handleError = () => {
+    toast.error(
+      intl.formatMessage({
+        defaultMessage: "Error: could not update applicant settings",
+        id: "J1YiBm",
+        description:
+          "Message displayed when an error occurs while updating applicant settings.",
+      }),
+    );
+  };
+
+  const handleFormSubmit: SubmitHandler<FormValues> = async (
+    values: FormValues,
+  ) => {
+    const data = formValuesToData(values);
+    await executeMutation(data)
+      .then((result) => {
+        if (result.data?.updateIgnoredNotifications) {
+          toast.success(
+            intl.formatMessage({
+              defaultMessage: "Updated applicant settings",
+              id: "zV8YQ1",
+              description:
+                "Message displayed when applicant updated their settings.",
+            }),
+          );
+        } else {
+          handleError();
+        }
+      })
+      .catch(() => {
+        handleError();
+      });
+  };
+
+  const formFields = [
+    {
+      id: "systemMessages",
+      name: "systemMessages",
+      legend: intl.formatMessage({
+        defaultMessage: "System Messages",
```

Comment:
```suggestion
        defaultMessage: "System messages",
```

Reply:
Fixed in [ae0068d](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/ae0068d800cd60af58d3a8ed31073fbd0a3e5bdc)